API.md

```
@@ -0,0 +1,398 @@
+curl "http://127.0.0.1:8000/v1/get_block_info?chain_id=1&number_or_hash=19000000&include_transactions=true"
+```
+
+### Transaction Tools
+
+#### Get Transaction Info (`get_transaction_info`)
+
+Gets comprehensive information for a single transaction.
+
+`GET /v1/get_transaction_info`
+
+**Parameters**
```

markdownlint-cli2 (0.17.2)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)